### ISSUE TITLE
New video

### ISSUE THREAD

**Issue by Ilyaskand** [dekrypted/discord-image-logger#95]
Make a new youtube video or something for tutorial because its hard to follow the intructions

**Comment by AR9NS** [comment]
it's common sense

**Comment by dekrypted** [comment]
> Make a new youtube video or something for tutorial because its hard to follow the intructions

if u cant follow the instructions u probably shouldn't be fucking with this...

also why do u think theres no video? why would i risk getting my channel terminated because ur too stupid to follow the instructions? itd just get removed again anyway
